pyaedt/edb_core/materials.py

```diff
@@ -788,3 +791,122 @@ def get_property_by_material_name(self, property_name, material_name):
                 else:
                     return property_box.ToDouble()
         return False
+
+    def load_material_from_aedt(self):
```

Comment:
@svandenb-dev, I suggest to rename the method to 

`def load_sys_library(self):`